### DISCUSSION TITLE
Multiple Errors & Exceptions (Bugs) that i found

### DISCUSSION THREAD

**Discussion by rahilhussain** [TheAlphamerc/flutter_twitter_clone#236] [General]
@TheAlphamerc   thank you.  If you provide the solutions for following problems it'll greately helpful for the people who're using your repository.Please Look into following queries.

These are three issues that I've identified,if possible please suggesest solutions to following queries:
1)Comment glitch -> after commenting on a tweet, it's getting added to database in backend but loading infinitely on frontend.
![image](https://user-images.githubusercontent.com/53215621/148647148-cc8dc484-0241-4e01-bd91-0101dd5edb44.png)

2)Multiple sign-in with google error -> if i sign in with another google account,I'm unable to follow any one,if i try i'm getting the following error given in screenshot. tweets are not loading at all.

![image](https://user-images.githubusercontent.com/53215621/148647249-003a8c81-e287-4368-adac-b6fcbe193f03.png)

3)Share button -> Share link/Share on social media button not working,if i try i'm getting the following error given in screenshot.

![image](https://user-images.githubusercontent.com/53215621/148647440-e72188ef-8538-4bbe-965f-8d50c17bc124.png)

**Comment by TheAlphamerc** [comment]
@rahilhussain do check my comment here -> https://github.com/TheAlphamerc/flutter_twitter_clone/issues/234#issuecomment-1008228023
